### PR TITLE
justify: init at unstable-2022-03-19

### PR DESCRIPTION
**Pull request by xfnw**
###### Description of changes
add package justify

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/text/justify/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  pname = "justify";
+  version = "unstable-2022-03-19";
+
+  src = fetchgit {
+    url = "https://tildegit.org/jns/justify.git";
+    rev = "0d397c20ed921c8e091bf18e548d174e15810e62";
+    sha256 = "sha256-406OhJt2Ila/LIhfqJXhbFqFxJJiRyMVI4/VK8Y43kc=";
+  };
+
+  buildInputs = [ cmake ];
```
Comment (SuperSandro2000) [inline pkgs/tools/text/justify/default.nix:13]:
```suggestion
  nativeBuildInputs = [ cmake ];
```

File: pkgs/tools/text/justify/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{lib, stdenv, fetchgit, cmake}:
+
+stdenv.mkDerivation {
+  pname = "justify";
+  version = "unstable-2022-03-19";
+
+  src = fetchgit {
```
Comment (SuperSandro2000) [inline pkgs/tools/text/justify/default.nix:7]: Please use fetchGitea.

**Review by IvarWithoutBones — COMMENTED** [review]
I have a few suggestions :blush:

File: pkgs/tools/text/justify/default.nix [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    sha256 = "sha256-406OhJt2Ila/LIhfqJXhbFqFxJJiRyMVI4/VK8Y43kc=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  installPhase = ''
+    install -D justify $out/bin/justify
+  '';
+
+  meta = with lib; {
```
Comment (IvarWithoutBones) [inline pkgs/tools/text/justify/default.nix:25]: Looks like I missed this before, could you add `meta.maintainers`? All new packages need to have a maintainer set.

**Review by IvarWithoutBones — CHANGES_REQUESTED** [review]
One last comment, other than that this looks good to me 👍

**Pull request merged**